sentry-jul/build.gradle.kts

```diff
@@ -71,7 +71,7 @@ tasks {
 buildConfig {
     useJavaOutput()
     packageName("io.sentry.logback")
-    buildConfigField("String", "SENTRY_JUL_SDK_NAME", "\"${Config.Sentry.SENTRY_LOGBACK_SDK_NAME}\"")
+    buildConfigField("String", "SENTRY_JUL_SDK_NAME", "\"${Config.Sentry.SENTRY_JUL_SDK_NAME}\"")
```

Comment:
this alone won't fix the problem, see the issue description, specifically this https://github.com/getsentry/sentry-java/blob/main/sentry-jul/src/main/java/io/sentry/jul/SentryHandler.java#L8

Reply:
Good point. Fixed